CHANGELOG.md

```diff
@@ -3,10 +3,16 @@ All notable changes to the "azure-appservice" extension will be documented in th
 
 ## 0.5.0 - 2017-12-15
 ### Added
-- Change Deployment Source for apps and slots
-- Deploy from Explorer
-- Auto-detects deployment method
-- Select configuration source when adding deployment slots
+- Configure web app's deployment source from the explorer
+- Deploy from Explorer with a button
+- Deploy projects immediately after web app creation
+- Zip deployment configuration
+- Create deployment slots from configuration sources
+- Show current running state of web apps
+- Improve resource deployment scripts
+- Leverage npm package "vscode-azureextensionui"
```

Comment:
Do we need to list engineering work here? The results of this work are worth mentioning though, like "load more", "creating..." placeholder, cached nodes etc.

Reply:
Yeah, I felt the same way.  

I just wasn't sure which new features to include.  For example, cached nodes could confuse the users into thinking that the nodes are cached between sessions, causing confusion (complaints) when the extension is super slow on each start up 😆